scripts/check_frontend_coverage.py

```diff
@@ -85,7 +85,6 @@
     'concept-card.directive.ts',
     'ConceptCardObjectFactory.ts',
     'context.service.ts',
-    'continue-button.directive.ts',
```

Comment:
Add a test file for this directive*

/cc @srijanreddy98, Can you please help Eric to test the component in/out.

Reply:
Yup, I will do that.